### DISCUSSION TITLE
Module.css not loading styles NEXT js

### DISCUSSION THREAD

**Discussion by andres-dejesus** [vercel/next.js#36866] [Help] [answered]
Hi, 

I'm trying apply module.css to one of my components according to this guide https://nextjs.org/learn/basics/assets-metadata-css/layout-component.

However only the main tag gets the css styling and not the class live-stream:

[Chrome elements picture](https://i.stack.imgur.com/LaoK0.png)
[Chrome elements picture2](https://i.stack.imgur.com/0KZFM.png
)
```/*/components/Livestream/App.js*/

import styles from './App.module.css';
...
return (
<div className={styles.container}>
 <main>
    <div className='live-stream'>
...
export default App;

/*/components/Livestream/App.module.css*/

.container main {
  align-items: center;
  display: flex;
  height: 100vh;
  justify-content: center;
}

.container main .live-stream {
  background-color: #000615;
  display: flex;
  height: 100%;
  overflow: hidden;
  position: relative;
  width: 100%;
}`

**Comment by icyJoseph** [comment, marked as answer]
Hi,

This is a CSS Modules usage issue.

Here's a good guide: https://www.javascriptstuff.com/css-modules-by-example/#example-4-nested-class-names

And in your case, this should work:

```tsx
import styles from "./App.module.css";

const LiveStream = () => {
  return (
    <div className={styles.container}>
      <main>
        <div className={styles["live-stream"]}></div>
      </main>
    </div>
  );
};

export default LiveStream;
```

Some people will say that `styles["live-stream"]` is not so nice, so you could change the CSS selector to:

```css
.container main .liveStream 
```

And then do `styles.liveStream`. Here's a guide for naming: https://github.com/css-modules/css-modules#naming

**Reply by andres-dejesus** [reply]
So you are right that works, but for my case with muiltiple classes, it does not:

`<div className={showMembers ? styles['separator showMembers'] : styles.separator}>`
` <div className={showMembers ? styles.separator.showMembers : styles.separator}>`

```
.container main > .liveStream > .separator.showMembers {
     width: 34rem;
}
```

**Reply by icyJoseph** [reply]
You need to join them, like so, `${styles.someClass} ${otherClass}`, these are all CSS modules quirks :)

**Reply by andres-dejesus** [reply]
fixed:

```<div className={showMembers ? `${styles.separator} ${styles.showMembers}` : styles.separator}>```

**Reply by icyJoseph** [reply]
Awesome! Don't forget to mark as answered and ask away if you have more doubts.
